mycpp/leaky_mylib.h

```diff
@@ -125,12 +125,28 @@ class Writer : public Obj {
   virtual bool isatty() = 0;
 };
 
+class Buf {
+ public:
+  Buf(char* data, int len) : data_(data), len_(len) {}
+  explicit operator bool() { return len_ == 0; }
```

Comment:
I prefer not to use any operator overloading (and I think that's in the Google style guide too)

This could be `IsEmpty()` ?

Reply:
done